### DISCUSSION TITLE
https://adyanth.site/posts/migration-compose-k8s/cloudflare-tunnel-operator-architecture/

### DISCUSSION THREAD

**Discussion by giscus** [adyanth/blog-comments#16] [General]
# https://adyanth.site/posts/migration-compose-k8s/cloudflare-tunnel-operator-architecture/

A bird's eye view of the Cloudflare Tunnel Operator.

https://adyanth.site/posts/migration-compose-k8s/cloudflare-tunnel-operator-architecture/

**Comment by munibshah** [comment]
This is great Adyanth. Keep it up!
